tools/picker/src/index.js

```diff
@@ -6,6 +6,7 @@ import getCategoriesInCategory from './queries/categories.graphql.js';
 import getProductsInCategory from './queries/products.graphql.js';
```

Comment:
Please make sure to run a build for the picker and add the build artifacts to git before merging.

Reply:
hey @herzog31 , what do you mean to add the build artifacts to git ?

Reply:
@mitre10 Run `npm run build` in `tools/picker` and add the files in `dist` to git.